.github/workflows/builds.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
               sv_repo=$(echo "${body}" | awk '/^suricata-verify-repo/ { print $2 }')
               sv_branch=$(echo "${body}" | awk '/^suricata-verify-branch/ { print $2 }')
-              sv_pr=$(echo "${body}" | awk '/^suricata-verify-pr/ { print $2 }')
+              sv_pr=$(echo "${body}" | awk '/^suricata-verify-pr/ { print $2 }' | tr -dc '0-9')
```

Comment:
This commit is no longer needed -- see d2c8c9f58e6608521afaf4484292d607e435f8c4

Reply:
Done :-)